### DISCUSSION TITLE
Moderation by user account?

### DISCUSSION THREAD

**Discussion by pearl-did-it** [umputun/remark42#1365] [Q&A]
Is it possible to configure remark42 for a site/app with multiple users who all post and share their own content?

- we'd need individual users to be able to moderate the comments posted to content associated with their account - approve, reject, lock threads, etc.
- users logged into the site/app need to be able to post or reply to comments using their account credentials - we're using google and apple sign-in but users can also create an account in our system, which we'd also want to work for comment name and notifications.

Apologize if I've missed these details elsewhere. Thanks for your help!

**Comment by paskal** [comment]
Apologies for missing answering that one right away: it's not possible currently, and we don't have a plan to implement it yet. Let's see if there would be more requestors for that feature.
